internal/handlers/billing/billing.go

```diff
@@ -0,0 +1,21 @@
+package billing
+
+import (
+	"github.com/glasskube/distr/internal/handlers"
```

Comment:
I don't like that `handlers/billing` imports from `handlers`. I think it would be better if this package had its own instance of the middleware